### ISSUE TITLE
minor issue and suggestion

### ISSUE THREAD

**Issue by leandro2222** [refik/rotate-sphere#2]
**Minor issue**: for me, it only worked when I replaced:
```
<script src="//ajax.googleapis....
```
by
```
<script src="http://ajax.googleapis....
```

**Suggestion**: It would be **VERY COOL** if you add an option to display the colorful XYZ axis like you did in the [blog post](https://medium.com/@refik/a-journey-and-a-method-for-drawing-spheres-5b24246ca479):
![colorful axis](https://miro.medium.com/v2/resize:fit:638/1*qoznb8N6zMqfPke1WlZvNw.gif)

see ya (=
